tests/testsuite/build_script_env.rs

```diff
@@ -237,6 +264,7 @@ fn build_script_sees_cfg_target_feature() {
 /// what the value of `RUSTFLAGS` should be in this case. We chose to give a
 /// warning. However, no matter what we do, it's important that build scripts
 /// and rustc see a consistent picture
+#[allow(deprecated)]
```

Comment:
Highlighting this for better tracking

Reply:
Sorry, I don't know how to emphasize this to make it more obvious (does that mean add notes?) . Can you go into more detail or examples?

Reply:
We had a race condition.  I was posting at the same time as you.  What you did was what I would recommend: start discussions on topics that you know reviewers need to look at.